common/FileUtils.java

```diff
@@ -118,6 +118,17 @@ public boolean fileOrFolderExists(String path) {
             || Files.isDirectory(fileSystem.getPath(expandHomeDirectory(path)));
   }
 
+  /** Checks if folder exists. Returns false for files. */
+  public boolean folderEmptyOrNotExists(String path) {
+    if (fileExists(path)) {
+      return false;
+    }
+    if (!folderExists(path)) {
+      return true;
+    }
+    return getFiles(path).isEmpty();
```

Comment:
s/getFiles(path)/listContents(path)

The method has already been renamed. And it throws IOException.